hugo/content/exporter/_index.md

```diff
@@ -4,26 +4,27 @@ draft: false
 weight: 1
 ---
 
-The exporters below can be set up on any Linux-based system, but the instructions below use RHEL/CentOS 6 or 7. Version 7 or higher is recommended if possible.
+The Linux instructions below use RHEL, but any Linux-based system should work. [Crunchy Data](https://www.crunchydata.com) customers can obtain Linux and Windows Server 2012R2 packages through the [Crunchy Customer Portal](https://access.crunchydata.com/).
 
 - [Installation](#installation)
+   - [Upgrading](#upgrading)
+   - [Linux RPMs](#linux-rpms)
+   - [Linux non-RPMs](#linux-non-rpms)
+   - [Windows packages](#windows-packages)
 - [Setup](#setup)
-   - [RHEL / CentOS 7](#setup-on-rhelcentos-7)
+   - [RHEL / CentOS 7 (preferred)](#setup-on-rhelcentos-7)
```

Comment:
Both 7 and 6 links are broken

Reply:
Add "(preferred)" in section heading line # 113 ?

Reply:
done